solvers/PGD.py

```diff
@@ -16,15 +18,16 @@ class Solver(BaseSolver):
     install_cmd = 'conda'
 
     # We need blas devel to get the include file for BLAS/LAPACK operations
-    requirements = ["blas-devel", 'pip:prox-tv']
+    requirements = ['pip:prox-tv', 'pip:numba', 'pip:numpy']
```

Comment:
`numpy` is already a dependance of `benchopt` I think. And we try to favor `conda` install for packages so if it exists in conda-forge, we try to use  it.
```suggestion
    requirements = ['pip:prox-tv', 'numba']
```